backend/src/main/java/com/htecgroup/skynest/model/request/UserEditRequest.java

```diff
@@ -30,4 +30,8 @@ public class UserEditRequest {
   @NotBlank(message = "cannot be null or empty")
   @Size(max = 254, message = "length cannot be over 254")
   private String address;
+
+  @NotBlank(message = "cannot be null or empty")
+  @Size(max = 100, message = "length cannot be over 100")
+  private String positionInCompany;
```

Comment:
I think it's 255 in the db

though it doesn't really matter

Reply:
Let's keep it consistent then